### PR TITLE
[Task #29] Add tables of content to markdown files

### PR DESCRIPTION
**Pull request by piotrekg35**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Table of contents
```
Comment (Laronk) [inline CONTRIBUTING.md:1]: ToC should be moved after "Contributing" section and the "Contributing section should be removed from the ToC.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+>### 1. [Contributing](#contributing)
+>### 2. [General rules](#general-rules)
+>### 3. [Git Workflow](#git-workflow)
+
```
Comment (Laronk) [inline CONTRIBUTING.md:6]: Each of level two and three headers should be included in the ToC with correct indentation.
Level three headers in "General rules section" should be excluded from ToC.

File: AGILE_WORKFLOW.md [hunk 3]
```diff
@@ -1,4 +1,12 @@
 # Agile workflow in Meshi project development
+## Table of contents
+>### 1. [Issue types](#issue-types)
+>### 2. [Writing Issues](#writing-issues)
+>### 3. [Team meetings](#team-meetings)
+>### 4. [Task estimating](#task-estimating-**Story-Points**)
+>### 5. [Additional definitions](#additional-definitions)
+
```
Comment (Laronk) [inline AGILE_WORKFLOW.md:8]: Each header should be included in the ToC with correct indentation.

**Review by Laronk — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (piotrekg35) [reply CONTRIBUTING.md]: Fixed. See 0a0cca534164326174e4ff38bbc4717b56003dfc

File: AGILE_WORKFLOW.md (hunk 3 above)
Comment (piotrekg35) [reply AGILE_WORKFLOW.md:16]: Including every header would make ToC look weird. See what I've changed 0a0cca534164326174e4ff38bbc4717b56003dfc and let me know if it's ok.

File: CONTRIBUTING.md (hunk 2 above)
Comment (piotrekg35) [reply CONTRIBUTING.md]: Including every header would make ToC look weird. See what I've changed 0a0cca534164326174e4ff38bbc4717b56003dfc and let me know if it's ok.

File: AGILE_WORKFLOW.md (hunk 3 above)
Comment (piotrekg35) [reply AGILE_WORKFLOW.md:16]: Fixed. See 24f064cd6cdf8a3f991a0c560a6cc2e21d4c9f2e

File: CONTRIBUTING.md (hunk 2 above)
Comment (Laronk) [reply CONTRIBUTING.md]: Yeah, you are right. I like the solution you provided.

File: AGILE_WORKFLOW.md (hunk 3 above)
Comment (Laronk) [reply AGILE_WORKFLOW.md:16]: Nice.

**Review by Laronk — APPROVED** [review]
Now it looks good.

**Review by danielhenel — COMMENTED** [review]
Page scroll doesn't work:

CONTRIBUTING.md:
-Language
-Issues
-Commits
-Pull Requests

AGILE_WORKFLOW.md:
-KISS rule
-Task estimating

Please fix this

**Pull request merged**